vehicle/SUV.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SUV_H
+#define SUV_H
+
+#include "ICar.h"
+
+template <typename EngineType, typename GearType>
```

Comment:
same here: why not simply use interface?

Reply:
template removed